tyr/servers/nginx/server.py

```diff
@@ -35,5 +38,19 @@ def __init__(self, group=None, server_type=None, instance_type=None,
                                           chef_path, subnet_id, dns_zones)
 
     def configure(self):
-
+        super(NginxServer, self).establish_logger()
```

Comment:
If `establish_logger` isn't being called automatically through your usage of `NginxServer` (as is the case with the `NginxCluster` above) you should probably call `establish_logger` after creating the server instance instead of adding it to `configure`.